aqua/qt6/files/patch-qtbase-macdeployqt.diff

```diff
@@ -0,0 +1,33 @@
+From c66dab56b20a47062c74fa6ecf8ea438bdc07b24 Mon Sep 17 00:00:00 2001
+From: Jonas Kvinge <jonas@jkvinge.net>
+Date: Tue, 26 Sep 2023 19:02:47 +0200
+Subject: [PATCH] macdeployqt: Also look for app libraries with .so extension
+MIME-Version: 1.0
+Content-Type: text/plain; charset=utf8
+Content-Transfer-Encoding: 8bit
+
+Fixes deploying glib-networking gio modules which uses .so file
+extension.
+
+Change-Id: I6b4c4e9c3bb5745ffa33d7e83c5853a9372f1ca6
+Reviewed-by: Tor Arne VestbÃ¸ <tor.arne.vestbo@qt.io>
+---
+ src/tools/macdeployqt/shared/shared.cpp | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/tools/macdeployqt/shared/shared.cpp b/src/tools/macdeployqt/shared/shared.cpp
+index 0671ebf2f32f..413f8413f69f 100644
+--- a/src/tools/macdeployqt/shared/shared.cpp
++++ b/src/tools/macdeployqt/shared/shared.cpp
```

Comment:
`patch -p0` is used by default, so I think the prefix needs to be removed (see other qt6 patches).

```suggestion
--- src/tools/macdeployqt/shared/shared.cpp
+++ src/tools/macdeployqt/shared/shared.cpp
```